testutil/ciao-down/workload.go

```diff
@@ -76,6 +81,40 @@ func (wkld *workload) save(ws *workspace) error {
 }
 
 func loadWorkloadData(ws *workspace, workloadName string) ([]byte, error) {
+	var absPath string
+	u, err := url.Parse(workloadName)
+	if err == nil {
+		switch u.Scheme {
+		case "http", "https":
+			workloadFile, err := ioutil.TempFile("/tmp", ".workload")
+			if err != nil {
+				break
+			}
+
+			absPath = workloadFile.Name()
+			defer os.Remove(absPath)
+
+			// 10 seconds should be enough to download the workload file
+			ctx, cancelFunc := context.WithTimeout(context.Background(), 10*time.Second)
+			defer cancelFunc()
+			err = getFile(ctx, workloadName, workloadFile, downloadProgress)
+			if err != nil {
+				return nil, fmt.Errorf("Unable download workload file from %s: %v", workloadName, err)
+			}
+		case "file":
+			absPath = path.Join("/", u.Host, u.Path)
+		default:
+			absPath, _ = filepath.Abs(u.Path)
+		}
+	}
+
+	if strings.Compare(absPath, "") != 0 {
+		wkld, err := ioutil.ReadFile(absPath)
```

Comment:
So here's a question.  When we've downloaded the file or read it from another location on our disk, do we want to store the file in our ~/.ciao-down/.workloads directory so we can use it to create other instances without having to download it again?  This is not what the code currently does, and maybe that is fine, but I wanted to raise the question.

@dlespiau, @rbradford any preferences here?